### PR TITLE
 BACKLOG-20114: Update header

### PR DESCRIPTION
**Pull request by cedmail**
## JIRA

https://jira.jahia.org/browse/ BACKLOG-20114

## Description

If nothing is selected all entires in header are related to main path
If there is only one node selected all entires in header are related to this path
If there is more than one node selected all entires in header are related to the selected path

Open in live shows only if node is available in live and its closest renderable parent is also in live and published

## Tests

The following are included in this PR
- [ ] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [ ] Performance
- [ ] Migration
- [ ] Code maintainability

## Documentation

- [ ] Inline documentation
- [ ] Internal Documentation (wiki)
- [ ] User-facing Documentation

### CODE REVIEW

File: src/javascript/JContent/ContentRoute/ContentLayout/ContentTable/ContentTable.jsx [hunk 1]
```diff
@@ -163,6 +163,7 @@ export const ContentTable = ({rows, isContentNotFound, totalCount, isLoading, is
                             prepareRow(row);
                             return (
                                 <Row key={'row' + row.id}
+                                     cl
```
Comment (gflores-jahia) [inline src/javascript/JContent/ContentRoute/ContentLayout/ContentTable/ContentTable.jsx:166]: typo?

File: tests/provisioning-manifest-build.yml [hunk 2]
```diff
@@ -1,2 +1,10 @@
 - addMavenRepository: 'https://store.jahia.com/nexus/content/repositories/jahia-public-app-store@id=JahiaStore'
 - addMavenRepository: 'https://devtools.jahia.com/nexus/content/groups/public/'
+
+- addMavenRepository: 'https://devtools.jahia.com/nexus/content/groups/public/@snapshots@noreleases'
+- installBundle:
+    - 'mvn:org.jahia.modules/jcontent'
```
Comment (gflores-jahia) [inline tests/provisioning-manifest-build.yml:6]: if this is build provisioning, shouldn't we be using the recently built version, not the one from maven (unless they're the same in this case)?

File: src/javascript/JContent/ContentRoute/ContentLayout/ContentTable/ContentTable.jsx (hunk 1 above)
Comment (cedmail) [reply src/javascript/JContent/ContentRoute/ContentLayout/ContentTable/ContentTable.jsx:166]: indeed

File: tests/provisioning-manifest-build.yml (hunk 2 above)
Comment (cedmail) [reply tests/provisioning-manifest-build.yml:6]: Should not have been committed, and it was wrong

**Review by gflores-jahia — APPROVED** [review]

**Pull request merged**